packages/gamut/src/ContentContainer/__tests__/ContentContainer-test.tsx

```diff
@@ -1,14 +1,17 @@
-import { mount } from 'enzyme';
+import { contentWidths } from '@codecademy/gamut-styles';
+import { render } from '@testing-library/react';
 import React from 'react';
 
 import { ContentContainer } from '..';
 
-describe('ContentContainer', () => {
-  it('contains a wide container class when the "wide" prop is passed in as true', () => {
-    const wrapper = mount(<ContentContainer wide />);
+type mockRenderProp = Record<'size', 'medium' | 'wide'>;
 
-    const rootClassNames = wrapper.childAt(0).prop('className');
+const mockRender = (props: mockRenderProp) =>
+  render(<ContentContainer {...props} />);
```

Comment:
We can get use gamut tests here to ensure theme provider is around.


```suggestion
import { setupRtl } from "@codecademy/gamut-tests";

const renderView = setupRtl(ContentContainer);

describe('ContentContainer', () => {
  it('has maxWidth of contentWidths.max when size is medium', () => {
    const { view } = renderView({ size: 'medium' });
    expect(view.firstChild).toHaveStyle(`maxWidth: ${contentWidths.max}`);
  });
});
```